modules/openapi-generator/src/main/resources/Java/libraries/feign/ApiClient.mustache

```diff
@@ -55,6 +64,10 @@ public class ApiClient {
                 .client(new OkHttpClient())
                 .encoder(new FormEncoder(new JacksonEncoder(objectMapper)))
                 .decoder(new JacksonDecoder(objectMapper))
+                {{#hasOAuthMethods}}
+                .errorDecoder(new ApiErrorDecoder())
+                .retryer(new Retryer.Default(0, 0, 2))
```

Comment:
probably make sense to extract this to instance variable or protected method so that end-user can change/override this values

Reply:
I believe the end-user is already able to customize the feignBuilder. Either by using the getFeignBuilder that retrieves the default builder, or he can override it with the setFeignBuilder.

Do you think I should add a specific method to disable/enable the retryer?

Reply:
@hugo-ma-alves My take is to try with what you've so far. Thanks for the PR.

@borsch thanks for the review.